libcircle.spec

```
@@ -0,0 +1,158 @@
+Name:    libcircle
+Version: 0.2.1
```

Is this version new enough for our needs?  Even if so, is this 0.2.1 RC1 the upstream version?  Have they not released an 0.2.1 GA?  If the upstream version is actually RC1 then the `rc1` needs to go in the version as such:

```suggestion
Version: 0.2.1~rc1
```

---

I am just following what Fedora/30 did.
Looks like the versions go from 0.2.1-rc1 to 0.3.0 and 0.2.1-rc1 is what Leap-15.2 provides.

---

Do we know that this 0.2.1~rc1 version, not 0.3, is new enough to meet CORCI-985's requirements though?

That the Fedora spec is putting the `rc1` in the release is just wrong.  If an 0.2.1 GA were ever released, it would not update the `rc1` RPM that they are producing.  It's a bug in their packaging.  Perhaps the packager doesn't understand how to indicate pre-release versions in spec files.

---

CORCI-985 is specifying 0.3 release.   Dropping the 0.2.1~rc1 on the next push.
Just doing CentOS 7 for this pass.